wordpress-mobile/WordPress-Android
# Add Optimizely A/B Testing framework

Fixes #3988 

To test:

1. First update your `gradle.properties` to contain `wp.optimizely_token`
2. When you get to the LoginActivity when building off this branch, you will see "Add self-hosted site" become "Not on WordPress.com?"

![screenshot_20160425-154158](https://cloud.githubusercontent.com/assets/1158819/14801593/49bb147c-0aff-11e6-8dc0-cf0de4d663c4.png)

Needs review: @maxme and @daniloercoli (for the Tracks events)
